packages/generated/medusa-client-types/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@medusajs/medusa-client-types",
```

Comment:
very nit(non-blocking): 
```suggestion
  "name": "@medusajs/client-types",
```

Reply:
Good idea, done in 5ab163b2640689e323c9b33c7827a577c5272321